### DISCUSSION TITLE
Make all fields readonly

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#12026] [Help] [answered]
In my workkflow app, I have some steps where all form fields must be readonly.
I find it heavy to put a condition on each field... 
Do you see a smarter way to easily make a readonly form, without having to duplicate the screen ?

**Comment by NicolasFabre81** [comment]
Finally I chose to duplicate the screen, but feel free to share better ideas !

**Reply by zedascouves** [reply, marked as answer]
@NicolasFabre81: In the form component, there's a "disabled" option. If you check it, all the fields in it will become non-editable, aka, read-only.
![image](https://github.com/Budibase/budibase/assets/594831/aacb25e6-5dc7-4366-99e2-6d8295937fbd)

**Reply by NicolasFabre81** [reply]
So great I've never noticed it ! Thanks !

**Reply by zedascouves** [reply]
Please mark the reply as the answer, so it can help more people in the future. :-)
